package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "questdk-plugins",
+  "type": "module",
   "scripts": {
     "create": "pnpm build && pnpm run -r --filter=@rabbitholegg/create-plugin create",
-    "build": "turbo run build --filter=!@rabbitholegg/questdk-plugin-project",
+    "build": "NODE_OPTIONS='--max-old-space-size=6096' turbo run build --filter=!@rabbitholegg/questdk-plugin-project,!@rabbitholegg/questdk-plugin-connext",
```

Comment:
Why excluding connext from the build step? Im a bit curious 🧐

Reply:
This doesn't actually work, I've gotta remove this line but for some reason connect is **eating** RAM during the build step